### ISSUE TITLE
Ability to add an author from the author merge page

### ISSUE THREAD

**Issue by RayBB** [internetarchive/openlibrary#10107]
### Proposal

As mentioned [in Slack](https://internetarchive.slack.com/archives/C0ETZV72L/p1732663264171859) I think it would be super handy if we copied the author box from the works page and were able to add an author to the author merge page that way.

Here's a very ugly screenshot of what it might look like.
![CleanShot 2024-12-02 at 16 50 46@2x](https://github.com/user-attachments/assets/b5d8d3d4-6457-40ff-89df-f55475432241)

### Justification

The problem is I have a tab addiction and like to open a lot of tabs when looking at authors to merge and it can be hard to get them all in one page that can be clicked. It would be nice to be able to just drop and author name or ID in the merge page and add it. 

We need approval from staff before we start working on this. It's really only applicable to librarians so may not be a very high priority.

#### Requirements Checklist

* [ ]

#### Related files

*

#### Stakeholders

*

<hr>

#### Instructions for Contributors

Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

**Comment by RayBB** [comment]
Closing in favor of https://github.com/internetarchive/openlibrary/issues/10113
